src/translation/translations/en.json

```diff
@@ -829,6 +829,9 @@
   },
   "voteProposalUi": {
     "activeUntilDate": "Active until: <Date>{{ date, dd MMM h:mm a }}</Date>",
+    "queuedUntilDate": "Queued until: <Date>{{ date, dd MMM h:mm a }}</Date>",
+    "canceledDate": "Canceled: <Date>{{ date, dd MMM h:mm a }}</Date>",
```

Comment:
Since we write "Cancelled" on the proposal already, let's write it the same way (with 2 L's).
<img width="876" alt="Screen Shot 2023-03-16 at 10 31 08" src="https://user-images.githubusercontent.com/44675210/225574564-05982473-2574-4266-8eb3-be89acb7a6ce.png">